packages/atomic/cypress/integration/facets/category-facet/category-facet.cypress.ts

```diff
@@ -224,13 +221,9 @@ describe('Category Facet Test Suites', () => {
       function setupGoDeeperLastLevel() {
         setupWithDefaultSettings();
         selectChildValueAt(canadaHierarchyIndex[0]);
-        cy.wait(TestFixture.interceptAliases.UA);
```

Comment:
Very nice that you have been able to remove all of those !

It means it's going in the right direction ! (Simpler tests).